src/Workspaces/Core/Portable/Experiments/IExperimentationService.cs

```diff
@@ -34,13 +38,11 @@ internal static class WellKnownExperimentNames
         public const string TargetTypedCompletionFilter = "Roslyn.TargetTypedCompletionFilter";
         public const string OOPServerGC = "Roslyn.OOPServerGC";
         public const string ImportsOnPasteDefaultEnabled = "Roslyn.ImportsOnPasteDefaultEnabled";
-        public const string LspTextSyncEnabled = "Roslyn.LspTextSyncEnabled";
```

Comment:
the code using these has since been deleted, so also removing the references here.